frontend/src/pages/org/archived-item-qa/archived-item-qa.ts

```diff
@@ -182,15 +188,33 @@ export class ArchivedItemQA extends TailwindElement {
     });
 
     window.addEventListener("message", this.onWindowMessage);
+    window.addEventListener("scroll", this.onWindowScroll);
   }
 
   disconnectedCallback(): void {
     super.disconnectedCallback();
+
     if (this.crawlData?.blobUrl) URL.revokeObjectURL(this.crawlData.blobUrl);
     if (this.qaData?.blobUrl) URL.revokeObjectURL(this.qaData.blobUrl);
+
     window.removeEventListener("message", this.onWindowMessage);
+    window.addEventListener("scroll", this.onWindowScroll);
   }
 
+  private scrollY = 0;
+  private readonly onWindowScroll = throttle(100)(() => {
+    // Set scroll snap only when scrolling down
```

Comment:
@emma-sg applying scroll snap was quick and seems to be working okay

Reply:
Ooh nice! clever